### DISCUSSION TITLE
How to target a ContentField with a different display mode in placement?

### DISCUSSION THREAD

**Discussion by gwonen** [OrchardCMS/OrchardCore#11257] [Q&A] [answered]
Apparently placement for ContentFields is tied to display mode, but I can't figure out how to tell fields with alternate display modes where to go. Documentation appears to be lacking.

e.g. `"TextField": [
    {
      "displayType": "Summary",
      "place": "Footer:1"
    }
  ]` no longer works when you switch the field's display mode to "Header". Assuming this is intentional, how do I tell this field where to go?

**Comment by ns8482e** [comment, marked as answer]
Use shape type`TextField_Display__[DisplayMode]` for placement

**Reply by gwonen** [reply]
Thanks. I had to put this in the placement.json file of my theme, didn't get picked up from the Placements feature.

**Reply by craftyweb** [reply]
Hi, I'm having the same issue with placement. I would like to be able to change the placement rules in the admin dashboard. Any idea why this is not working?
